### ISSUE TITLE
BE AWARE THAT SKYLINE IS STILL IN ALPHA BUILD.

### ISSUE THREAD

**Issue by sauradipdebnath** [skyline-emu/skyline#372]
Most of the games don't support graphics and it's not a bug and it's not implemented So As it's in a alpha build please don't make issues about graphics

**Comment by Bggggggggggg** [comment]
mt8243339@gmail.com

**Comment by Bggggggggggg** [comment]
Muhammad talha

**Comment by Bggggggggggg** [comment]
Ml NJ

**Comment by Bggggggggggg** [comment]
Frostbite

**Comment by Bggggggggggg** [comment]
Bgggggggggggg

**Comment by Bggggggggggg** [comment]
Bggggggggggg Frostbite

**Comment by Bggggggggggg** [comment]
De ururz z

**Comment by Sohailshah5050** [comment]
May i know list of games which work on skyline?

**Comment by marcoluc97** [comment]
Yeah i think maybe can be helpful to mitigate all the users without knowledge that are flooding askign the same thing, to just use the "playable" ones if there's a compatibility or tracking list of "games/homebrews" but is just a suggestion

**Comment by Flavio2314** [comment]
V

**Comment by Sondragon66** [comment]
Having some issues, The game screen is completely blank, But that just might be because it's an older version, is there the latest version I can download?

**Comment by NotAWeeb69** [comment]
> Having some issues, The game screen is completely blank, But that just might be because it's an older versio when I load in sonic mania, is there the latest version I can download?

This is a wrong place to ask that. Please refrain from opening issues to ask questions. Join the discord server instead

You can grab the newest ftx1 branch build of skyline in actions tab. Make sure its the ftx1 branch tho (not title-meta or settings) or else games wont work.
In general most mainline games do not work due to the fact skyline is still relatively new and is still actively being developed

**Comment by ShadowBlakc** [comment]
The recent emulator updates,
can not be installed on my phone,
F

**Comment by sauradipdebnath** [comment]
Pokemon games don't work

**Comment by sauradipdebnath** [comment]
BE AWARE THAT POKEMON GAMES DON'T WORK

**Comment by PixelyIon** [comment]
@sauradipdebnath I think we've warned you enough. A lot of these comments are entirely redundant and just spam the git feed. I'll be banning you from the repository for 7 days.
